op-node/p2p/node.go

```diff
@@ -146,6 +184,8 @@ func (n *NodeP2P) Close() error {
 		if err := n.host.Close(); err != nil {
 			result = multierror.Append(result, fmt.Errorf("failed to close p2p host cleanly: %w", err))
 		}
+		// TODO close sync loop
```

Comment:
I'm surprised semgrep doesn't seem to be flagging this. Does it need to be dealt with?